p3dx_motionplanner/P3DXPose[] goal
p3dx_motionplanner/P3DXPose[] state
p3dx_motionplanner/Obstacle[] obstacles
float32 current_time
